tex/latex/biblatex/lbx/french.lbx

```diff
@@ -114,7 +112,7 @@
 \DeclareBibliographyStrings{%
   bibliography     = {{Bibliographie}{Bibliographie}},
   references       = {{R\'ef\'erences}{R\'ef\'erences}},
-  shorthands       = {{Liste des sigles}{Sigles}},
+  shorthands       = {{Liste des abréviations}{Abréviations}},
```

Comment:
In this file we are still using macro escapes for non-ASCII chars, so instead of `é` we'd write `\'e` here.
```suggestion
  shorthands       = {{Liste des abr\'eviations}{Abr\'eviations}},
```

Reply:
Sorry…